### PR TITLE
Add Crescendo faucet

### PR DESCRIPTION
**Pull request by ianthpun**
Closes #???

## Description

---

- [ ] Targeted PR against `main` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Comment by ianthpun** [comment]
image
seems to be working

**Review by jribbink — APPROVED** [review]
lgtm

File: components/PageTitle.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import publicConfig from "lib/publicConfig"
 import Head from "next/head"
 
 export const BASE_HTML_TITLE = `Flow ${
-  publicConfig.network === TEST_NET ? "Testnet" : ""
+  publicConfig.network === TEST_NET ? "Testnet" : "Credecendo"
```
[the remark this answers is not in the document]
Comment (ianthpun) [reply components/PageTitle.tsx:6]: classic, thank you

**Comment by ianthpun** [comment]
@bthaile are you able to merge this in for me? We'll need to update the codeowners after this

**Pull request merged**